.github/workflows/physionet-build-test.yml

```diff
@@ -46,12 +46,9 @@ jobs:
           sudo -u postgres psql -c "create database physionet;" -U postgres
       - name: install and setup wfdb
         run: |
-          wget https://github.com/bemoody/wfdb/archive/10.6.2.tar.gz -O wfdb.tar.gz
+          wget https://github.com/bemoody/wfdb/archive/master.tar.gz -O wfdb.tar.gz
```

Comment:
it would be good to pin a version if possible, so that we can keep the tests in sync with whatever version we are running on our live servers.

Reply:
Thanks for the suggestion, I have brought this up in #1328 .